sale_stock_usability/sale_stock.py

```diff
@@ -2,11 +2,43 @@
 # @author Alexis de Lattre <alexis.delattre@akretion.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from odoo import models, fields
+from odoo import models, fields, api
 
 
 class SaleOrder(models.Model):
     _inherit = 'sale.order'
 
     warehouse_id = fields.Many2one(track_visibility='onchange')
     incoterm = fields.Many2one(track_visibility='onchange')
+    picking_status = fields.Selection([
```

Comment:
Deliverd

Reply:
is `deliverd` an english term ?

Reply:
Yes. according to google

Reply:
Incredible but real.

maybe `delivered` better known

https://www.deepl.com/translator#en/fr/delivered
https://www.deepl.com/translator#en/fr/deliverd